### PR TITLE
Set currentVote

### PR DESCRIPTION
**Pull request by wurmmi**
This is to fix issue #53 .

### CODE REVIEW

File: src/JukeBox.cpp [hunk 1]
```diff
@@ -113,19 +113,53 @@ TResult<vector<BaseTrack>> JukeBox::queryTracks(string const &searchPattern,
...
+  /* Set markers, if user has already voted for a track */
+  for (auto &queueElem : qs.adminQueue.tracks) {
+    for (auto const &votedElem : user.votes) {
+      if (queueElem.trackId == votedElem)
+        /* User has voted for this track */
+        queueElem.currentVote = true;
+      else
+        queueElem.currentVote = false;
+    }
+  }
```
Comment (janeson332) [inline src/JukeBox.cpp]: votes on admin queue tracks are not possible, so this is not necessary

**Comment by wurmmi** [comment]
can we test correct behavior before merge?

**Comment by wurmmi** [comment]
TODO: rename `.currentVote` --> `.userHasVoted`

**Comment by skaupper** [comment]
your logic was broken. but otherwise it seems fine now 👍

**Pull request merged**